File: app/src/main/java/ru/roscha_akademii/medialib/viewvideo/view/StubTextRendererOutput.java
package ru.roscha_akademii.medialib.viewvideo.view;

import android.util.Log;

import com.google.android.exoplayer2.text.Cue;
import com.google.android.exoplayer2.text.TextRenderer;

import java.util.List;

public class StubTextRendererOutput implements TextRenderer.Output {
    @Override
    public void onCues(List<Cue> cues) {
        if (cues != null) {
            for (Cue c : cues) {
                Log.d("happy", "cue " + c.text.toString());
            }
        }
    }
}

File: app/src/main/java/ru/roscha_akademii/medialib/viewvideo/view/StubTextRendererOutput.kt
package ru.roscha_akademii.medialib.viewvideo.view

import android.util.Log

import com.google.android.exoplayer2.text.Cue
import com.google.android.exoplayer2.text.TextRenderer

class StubTextRendererOutput : TextRenderer.Output {
    override fun onCues(cues: List<Cue>?) {
        if (cues != null) {
            for (c in cues) {
                Log.d("happy", "cue " + c.text.toString())
            }
        }
    }
}
